src/main/java/csula/cs4660/graphs/representations/AdjacencyList.java

```diff
@@ -18,46 +19,147 @@
 public class AdjacencyList implements Representation {
     private Map<Node, Collection<Edge>> adjacencyList;
 
-    public AdjacencyList(File file) {
+    public AdjacencyList(File file)
+    {
+        try {
+            adjacencyList = new HashMap<Node, Collection<Edge>>();
+            BufferedReader br = new BufferedReader(new FileReader(file));
+            String line = br.readLine();
+            int count = Integer.parseInt(line);
+            for (int i = 0; i < count; i++) {
+                adjacencyList.put(new Node(i), new ArrayList<Edge>());
+            }
+            line = br.readLine();
+            while (line != null) {
+                String[] data = line.split(":");
+                int[] idata = new int[data.length];
+                for (int j = 0; j < data.length; j++) {
+                    idata[j] = Integer.parseInt(data[j]);
+                }
+                ArrayList<Edge> temp = (ArrayList<Edge>) adjacencyList.get(new Node(idata[0]));
+                temp.add(new Edge(new Node(idata[0]), new Node(idata[1]), idata[2]));
+                adjacencyList.put(new Node(idata[0]), temp);
+                line = br.readLine();
+            }
+            br.close();
+        }catch(IOException e){}
     }
 
     public AdjacencyList() {
 
     }
 
+
     @Override
-    public boolean adjacent(Node x, Node y) {
+    public boolean adjacent(Node x, Node y)
+    {
+        ArrayList<Edge> list = (ArrayList<Edge>) adjacencyList.get(x);
```

Comment:
I don't see a point of casting to ArrayList. You should still be able to loop through collection.